app/scripts/modules/core/src/delivery/create/CreatePipelineModal.tsx

```diff
@@ -94,31 +99,62 @@ export class CreatePipelineModal extends React.Component<ICreatePipelineModalPro
       templates: [],
       existingNames: existingNames,
       command: {parallel: true, strategy: false, name: '', config: defaultConfig, template: null},
-      useTemplate: false,
+      useTemplate: SETTINGS.feature.pipelineTemplates,
+      useManagedTemplate: true,
+      loadingTemplateFromSource: false,
+      loadingTemplateFromSourceError: false,
+      templateSourceUrl: '',
     };
   }
 
   public submit(): void {
     const command = cloneDeep(this.state.command);
-    const config: Partial<IPipeline> = command.strategy ? this.getDefaultConfig() : command.config;
+    const pipelineConfig: Partial<IPipelineTemplateConfig> = command.strategy ? this.getDefaultConfig() : command.config;
 
-    config.name = command.name;
-    config.index = this.props.application.getDataSource('pipelineConfigs').data.length;
-    delete config.id;
+    pipelineConfig.name = command.name;
+    pipelineConfig.index = this.props.application.getDataSource('pipelineConfigs').data.length;
+    delete pipelineConfig.id;
 
     if (command.strategy) {
-      config.strategy = true;
-      config.limitConcurrent = false;
+      pipelineConfig.strategy = true;
+      pipelineConfig.limitConcurrent = false;
     }
+    if (pipelineConfig.type === 'templatedPipeline') {
+      delete pipelineConfig.config.pipeline.pipelineConfigId;
+      pipelineConfig.config.pipeline.name = command.name;
+    }
+
+    this.setState({submitting: true});
+    ReactInjector.pipelineConfigService.savePipeline(pipelineConfig as IPipeline)
+      .then(() => this.onSaveSuccess(pipelineConfig), this.onSaveFailure);
+  }
 
+  private submitPipelineTemplateConfig(): void {
+    const config: Partial<IPipelineTemplateConfig> = {
+      name: this.state.command.name,
+      application: this.props.application.name,
+      type: 'templatedPipeline',
+      limitConcurrent: true,
+      keepWaitingPipelines: false,
+      parallel: true,
+      executionEngine: 'v2',
```

Comment:
I'd just remove this field entirely. Orca should be the one to decide the default execution engine (which it does).